src/scene-entry-manager.js

```diff
@@ -230,6 +242,7 @@ export default class SceneEntryManager {
     });
 
     this.scene.addEventListener("pinned", e => {
+      if (!NAF.utils.isMine(e.detail.el)) return;
```

Comment:
what case causes this to need to be here? when you click the pin button, you should take ownership -- but maybe that's not working right?